modules/exploits/windows/misc/allmedia_1_6_buffer_overflow.rb

```diff
@@ -0,0 +1,89 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+# Author: Hejap Zairy
+# Date: 1.08.2022
+# Exploit Prof
+# Proof and Exploit:
+#image:https://i.imgur.com/yLrRR2t.png
+#video:https://streamable.com/x4i50c
+
+
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
```

Comment:
Looks like this was taken from old code. Metasploit4 is ancient now as we are on Metasploit 6. This is how this should be defined:

```suggestion
class MetasploitModule < Msf::Exploit::Remote
```